lib/Interpreter/CIFactory.cpp

```diff
@@ -316,14 +316,9 @@ namespace {
       }
 
     #if defined(__GLIBCXX__)
-      // Avoid '__float128 is not supported on this target' errors
```

Comment:
That whole block is specifically dealing with gcc on OS X and how to avoid the __float128 problem, so I don't see how the comment it is out of context. The fact it exists and a simple search of the file shows that `-std=c++11` is not in fact ever passed without it.

Can we step back. What are you trying to accomplish?

I can accept `-D __float128` is ugly and can easily add it on top later.
But you don't seem to actively use gcc on OS X, so why is it so important to change anything else in this block which I know solves actual problems and is commented explicitly stating what it is doing?